backend/server.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const app = express();
```

Comment:
1. Let's please move the app declaration after the require statements for better code organization and readability. 

2. Let's try to group imports based on their types. 

example: 

```
// external modules (npm installed)
const express = require("express");
const mongoose = require("mongoose");
const expressLayouts = require("express-ejs-layouts");

// routes 
const indexRouter = require("./routes/index");

```